playwright-e2e/lib/component/dsm/filters/sections/customize-view.ts

```diff
@@ -11,17 +11,27 @@ export class CustomizeView {
   public async selectColumns(columnsGroupName: string, columns: string[]): Promise<void> {
     this.activeColumnsGroup = columnsGroupName;
     await this.openColumnsGroup();
-    for (const column of columns) {
-      await this.selectColumn(column);
-    }
+    await this.select(columns);
     await this.closeColumnsGroup();
   }
 
-  private async selectColumn(columnName: string): Promise<void> {
+  public async deselectColumns(columnsGroupName: string, columns: string[]): Promise<void> {
+    this.activeColumnsGroup = columnsGroupName;
+    await this.openColumnsGroup();
+    await this.select(columns, true);
+    await this.closeColumnsGroup();
+  }
+
+  private async select(columns: string[], deselect = false): Promise<void> {
+    for (const column of columns) { await this.selectOrDeselect(column, deselect); }
+  }
+
+  private async selectOrDeselect(columnName: string, deselect = false): Promise<void> {
     const columnXPath = this.columnsGroupXPath + this.columnPathXPath(columnName);
     const column = this.page.locator(columnXPath);
 
-    !(await this.isChecked(column)) && (await this.page.locator(columnXPath).click());
+    !deselect ? !(await this.isChecked(column)) && await this.page.locator(columnXPath).click() :
+      await this.isChecked(column) && await this.page.locator(columnXPath).click();
```

Comment:
Since there are two calls each, I'd prefer to use if/else for clarity. What do you think?

Reply:
Otherwise, we should have nested if/else. Oki, I will do as you suggest.